### PR TITLE
Add Exchange reference

### PR DESCRIPTION
**Pull request by johndmulhausen**

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build failed...

```

> filecoin-docs@1.0.0 docs:build /home/runner/work/filecoin-docs/filecoin-docs
> npm run docs:links && vuepress build docs


> filecoin-docs@1.0.0 docs:links /home/runner/work/filecoin-docs/filecoin-docs
> vuepress check-md docs

(node:3186) UnhandledPromiseRejectionWarning: /home/runner/work/filecoin-docs/filecoin-docs/docs/.vuepress/config.js:200
            'exchanges',
            ^^^^^^^^^^^

SyntaxError: Unexpected string
    at wrapSafe (internal/modules/cjs/loader.js:1001:16)
    at Module._compile (internal/modules/cjs/loader.js:1049:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:93:18)
    at loadConfig (/home/runner/work/filecoin-docs/filecoin-docs/node_modules/@vuepress/core/lib/node/loadConfig.js:31:18)
    at App.resolveConfigAndInitialize (/home/runner/work/filecoin-docs/filecoin-docs/node_modules/@vuepress/core/lib/node/App.js:64:24)
    at App.process (/home/runner/work/filecoin-docs/filecoin-docs/node_modules/@vuepress/core/lib/node/App.js:103:16)
    at beforeParse (/home/runner/work/filecoin-docs/filecoin-docs/node_modules/vuepress/lib/handleUnknownCommand.js:117:19)
    at CLI (/home/runner/work/filecoin-docs/filecoin-docs/node_modules/vuepress/lib/util.js:22:24)
    at CAC.<anonymous> (/home/runner/work/filecoin-docs/filecoin-docs/node_modules/vuepress/lib/handleUnknownCommand.js:107:5)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:3186) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:3186) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
wait Extracting site metadata...
/home/runner/work/filecoin-docs/filecoin-docs/docs/.vuepress/config.js:200
            'exchanges',
            ^^^^^^^^^^^

SyntaxError: Unexpected string
    at wrapSafe (internal/modules/cjs/loader.js:1001:16)
    at Module._compile (internal/modules/cjs/loader.js:1049:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:93:18)
    at loadConfig (/home/runner/work/filecoin-docs/filecoin-docs/node_modules/@vuepress/core/lib/node/loadConfig.js:31:18)
    at App.resolveConfigAndInitialize (/home/runner/work/filecoin-docs/filecoin-docs/node_modules/@vuepress/core/lib/node/App.js:64:24)
    at App.process (/home/runner/work/filecoin-docs/filecoin-docs/node_modules/@vuepress/core/lib/node/App.js:103:16)
    at build (/home/runner/work/filecoin-docs/filecoin-docs/node_modules/@vuepress/core/lib/index.js:26:13)
    at /home/runner/work/filecoin-docs/filecoin-docs/node_modules/vuepress/lib/util.js:35:12
    at CAC.<anonymous> (/home/runner/work/filecoin-docs/filecoin-docs/node_modules/vuepress/lib/registerCoreCommands.js:62:25)
    at CAC.runMatchedCommand (/home/runner/work/filecoin-docs/filecoin-docs/node_modules/cac/dist/index.js:614:34)
    at CAC.parse (/home/runner/work/filecoin-docs/filecoin-docs/node_modules/cac/dist/index.js:541:12)
    at CLI (/home/runner/work/filecoin-docs/filecoin-docs/node_modules/vuepress/lib/util.js:23:7)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! filecoin-docs@1.0.0 docs:build: `npm run docs:links && vuepress build docs`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the filecoin-docs@1.0.0 docs:build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2021-11-04T15_11_03_226Z-debug.log
```

File: docs/reference/exchanges.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
+---
+title: Exchange Integration Guide
```
Comment (johnnymatthews) [inline docs/reference/exchanges.md:2]:
```suggestion
title: "Exchanges"
```

File: docs/reference/exchanges.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+description: 'Lorem ipsum.'
```
Comment (johnnymatthews) [inline docs/reference/exchanges.md:3]:
```suggestion
description: "Use this guide to help you list Filecoin on your exchange platform."
```

**Review by johnnymatthews — APPROVED** [review]

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

### `docs/build/powergate.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://godoc.org/github.com/textileio/powergate/api/client
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...les: Replication factor Miner selection Deal renewal and Repair # How to use...
                                             ^^^^
  ```
- Possible missing comma found.
  ```
  ...n, you need to fully sync from Filecoin MainNet which may take over a day. T...
                                             ^^^^^^^
  ```
- Possible missing comma found.
  ```
  ...the IPFS network, you don't need to run stage as the Powergate will automati...
                                             ^^^^^
  ```
### `docs/reference/exchanges.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://network.filecoin.io
- The following link returned HTTP status code 302 (Found):
  https://join.slack.com/t/filecoinproject/shared_invite/zt-mapdjgpy-AzltjeUC9Y0UQOLGjInZuQ
- The following link failed to connect:
  https://spec.filecoin.io.md
- Did you mean “uses”?
  ```
  # Filecoin Integration Guide Use this guide to help you list Filecoin (F...
                               ^^^
  ```
- Possible missing article found.
  ```
  ...stall the Lotus applications and launch node, basically you need to: Prepare...
                                             ^^^^
  ```
- Possible missing comma found.
  ```
  ...ain 3. Sync from scratch: in your lotus directory run lotus daemon 4. Sync f...
                                             ^^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...it to wait for the sync to be complete, Done! indicates that your node is fu...
                                             ^^^^
  ```
- This expression is usually spelled with a hyphen.
  ```
  ...ignatures and Messages Filecoin uses an account based model. There are 4 typ...
                                             ^^^^^^^^^^^^^
  ```
- Consider using a different verb to strengthen your wording.
  ```
  ...hort: f<address> === t<address>. Please make sure if you get a t address, it...
                                             ^^^^^^^^^
  ```
- Two determiners in a row. Choose either “the” or “a”.
  ```
  ...ated with each message, please refer to the a practical guide to Gas section...
                                             ^^^^^
  ```
- Possible missing comma found.
  ```
  ... necessary permissions. To find all CLI usage run lotus -h in your lotus fol...
                                             ^^^^^
  ```

**Comment by johnnymatthews** [comment]
Closes #1104.

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

### `docs/build/powergate.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://godoc.org/github.com/textileio/powergate/api/client
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...les: Replication factor Miner selection Deal renewal and Repair # How to use...
                                             ^^^^
  ```
- Possible missing comma found.
  ```
  ...n, you need to fully sync from Filecoin MainNet which may take over a day. T...
                                             ^^^^^^^
  ```
- Possible missing comma found.
  ```
  ...the IPFS network, you don't need to run stage as the Powergate will automati...
                                             ^^^^^
  ```
### `docs/reference/exchanges.md`:
- The following link failed to connect:
  https://spec.filecoin.io.md
- The following link returned HTTP status code 302 (Found):
  https://join.slack.com/t/filecoinproject/shared_invite/zt-mapdjgpy-AzltjeUC9Y0UQOLGjInZuQ
- The following link returned HTTP status code 301 (Moved Permanently):
  https://network.filecoin.io
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ... sync to be complete. Lotus will output Done! once your node is fully synced...
                                             ^^^^
  ```
- This expression is usually spelled with a hyphen.
  ```
  ...# Accounts and wallets Filecoin uses an account based model. There are 4 typ...
                                             ^^^^^^^^^^^^^
  ```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

### `docs/build/powergate.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://godoc.org/github.com/textileio/powergate/api/client
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...les: Replication factor Miner selection Deal renewal and Repair # How to use...
                                             ^^^^
  ```
- Possible missing comma found.
  ```
  ...n, you need to fully sync from Filecoin MainNet which may take over a day. T...
                                             ^^^^^^^
  ```
- Possible missing comma found.
  ```
  ...the IPFS network, you don't need to run stage as the Powergate will automati...
                                             ^^^^^
  ```
### `docs/reference/exchanges.md`:
- The following link failed to connect:
  https://spec.filecoin.io.md
- The following link returned HTTP status code 302 (Found):
  https://join.slack.com/t/filecoinproject/shared_invite/zt-mapdjgpy-AzltjeUC9Y0UQOLGjInZuQ
- The following link returned HTTP status code 301 (Moved Permanently):
  https://network.filecoin.io
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ... sync to be complete. Lotus will output Done! once your node is fully synced...
                                             ^^^^
  ```
- This expression is usually spelled with a hyphen.
  ```
  ...# Accounts and wallets Filecoin uses an account based model. There are 4 typ...
                                             ^^^^^^^^^^^^^
  ```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

### `docs/build/powergate.md`:
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...les: Replication factor Miner selection Deal renewal and Repair # How to use...
                                             ^^^^
  ```
- Possible missing comma found.
  ```
  ...n, you need to fully sync from Filecoin mainnet which may take over a day. T...
                                             ^^^^^^^
  ```
- Possible missing comma found.
  ```
  ...the IPFS network, you don't need to run stage as the Powergate will automati...
                                             ^^^^^
  ```
### `docs/reference/exchanges.md`:
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ... sync to be complete. Lotus will output Done! once your node is fully synced...
                                             ^^^^
  ```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

### `docs/build/powergate.md`:
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...les: Replication factor Miner selection Deal renewal and Repair # How to use...
                                             ^^^^
  ```
- Possible missing comma found.
  ```
  ...n, you need to fully sync from Filecoin mainnet which may take over a day. T...
                                             ^^^^^^^
  ```
- Possible missing comma found.
  ```
  ...the IPFS network, you don't need to run stage as the Powergate will automati...
                                             ^^^^^
  ```
### `docs/reference/exchanges.md`:
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ... sync to be complete. Lotus will output Done! once your node is fully synced...
                                             ^^^^
  ```

**Pull request merged**